airflow/auth/managers/fab/security_manager/modules/db.py

```diff
@@ -86,7 +86,7 @@ def create_db(self):
             if self.count_users() == 0 and self.auth_role_public != self.auth_role_admin:
                 log.warning(const.LOGMSG_WAR_SEC_NO_USER)
         except Exception as e:
-            log.error(const.LOGMSG_ERR_SEC_CREATE_DB.format(str(e)))
+            log.error(const.LOGMSG_ERR_SEC_CREATE_DB.format(e))
```

Comment:
Orthogontal, but I want to investigate fixing these eager formatting at some point.